Jenkinsfile.txt

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env groovy
+
+/* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
+buildPlugin()
```

Comment:
This is a great addition, but the file needs to be named `Jenkinsfile` rather than `Jenkinsfile.txt`